bitnami/memcached/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Infrastructure
 apiVersion: v2
-appVersion: 1.6.9
+appVersion: 1.7.0
```

Comment:
You need to bump the `version` not the `appVersion`

Reply:
ahh rights! thanks,  I will change it right away